src/spec.h

```diff
@@ -122,6 +122,17 @@ struct DocumentIndexer;
 extern dict *specDict_g;
 #define dictGetRef(he) ((StrongRef){dictGetVal(he)})
 
+
+#define DEBUG_INDEX_SCANNER_CODE_NEW 0
+#define DEBUG_INDEX_SCANNER_CODE_RUNNING 1
+#define DEBUG_INDEX_SCANNER_CODE_DONE 2
+#define DEBUG_INDEX_SCANNER_CODE_CANCELLED 3
+#define DEBUG_INDEX_SCANNER_CODE_PAUSED 4
+#define DEBUG_INDEX_SCANNER_CODE_RESUMED 5
+
+#define DEBUG_INDEX_SCANNER_STATUS_STRS \
+  { "NEW", "SCANNING", "DONE", "CANCELLED", "PAUSED", "RESUMED" }
+
```

Comment:
1. Consider using an `enum`
2. Consider moving all of these to `debug_commands.h` (status messages can maybe be on the c file only)
3. Consider adding a static assertion that the length of the status array is equal to the number of statuses (we have something similar for the coordinator's debug commands), so if someone adds/removes a status he will have to change the status messages. Alternatively, implement something similar to the query error messages array

Reply:
After we spoke, decided to keep it in spec.h
Changed to enum for cleaner and safer implementation.